### DISCUSSION TITLE
Error due to missing key prop in the .map function

### DISCUSSION THREAD

**Discussion by donacrisante** [neuefische/questions#654] [Web] [answered]
Hey there!
I am working on my first US, and I have written a map function for a dropdown menu. Now I always get the error that a key pop is missing. I have added a key, but the error is always there. So the Deployment on Vercel always fails. I have tried everything possible, but it just doesn't work. Maybe someone can help me? Thank you very much!

![Bildschirm­foto 2023-08-01 um 11 03 44](https://github.com/neuefische/questions/assets/128093782/9344638e-864f-4985-ae82-e19c162cf31a)

![Bildschirm­foto 2023-08-01 um 11 03 21](https://github.com/neuefische/questions/assets/128093782/0b7f5fc5-3d63-429e-a666-3ea518bcf604)

![Bildschirm­foto 2023-08-01 um 11 02 03](https://github.com/neuefische/questions/assets/128093782/455a97d4-9444-419e-b59a-855733340107)

**Comment by Kheirah** [comment, marked as answer]
Hey Dona 👋🏼 
I've looked into your project and everything works just fine.
The error message comes from a previous deployment. You already fixed the key prop 👍🏼 

Next time, though, please leave a link to your latest Pull Request. And instead of images, copy and paste the code you're referring to 😀

**Comment by donacrisante** [comment]
Ah that relieves me a bit because I really didn't know what else was wrong. But it irritates me a bit that the error keeps coming up, even when I redeploy.

**Reply by Kheirah** [reply]
You don't need to manually redeploy your app. Every time you add a commit and push it to your PR, Vercel creates an entirely new deployment of your changes. If you scroll down [this page](https://github.com/donacrisante/Capstone-project/pull/5) you will see that Vercel created several new deployments which all work.

**Comment by donacrisante** [comment]
Ok thanks!
